amundsen_application/static/js/components/common/Table/styles.scss

```diff
@@ -82,20 +84,22 @@ $row-bottom-border-color: $gray10;
   overflow: visible;
 
   &:first-child {
-    padding-left: $spacer-3;
+    padding-left: $spacer-3 !important;
```

Comment:
I wonder if there would be a way of avoiding this !important, maybe adding an extra class would be better.